src/me/mrCookieSlime/Slimefun/Setup/SlimefunSetup.java

```diff
@@ -1536,6 +1536,7 @@ else if (SlimefunManager.isItemSimiliar(current, SlimefunItems.PULVERIZED_ORE, t
 			@Override
 			public boolean onBlockBreak(BlockBreakEvent e, ItemStack item, int fortune, List<ItemStack> drops) {
 				if (SlimefunManager.isItemSimiliar(item, SlimefunItems.AUTO_SMELT_PICKAXE, true)) {
+					if (BlockStorage.hasBlockInfo(e.getBlock())) return false;
```

Comment:
This should return true.

true means that this Event was handled and should not be passed onto other Items.